src/ui/server/constants/declarations/get-latest-declarations/index.test.ts

```diff
@@ -25,6 +32,20 @@ describe('server/constants/declarations/get-latest-declarations', () => {
         ANTI_BRIBERY: ANTI_BRIBERY.VERSIONS[1],
         ANTI_BRIBERY_CODE_OF_CONDUCT: ANTI_BRIBERY_CODE_OF_CONDUCT.VERSIONS[1],
         ANTI_BRIBERY_EXPORTING_WITH_CODE_OF_CONDUCT: ANTI_BRIBERY_EXPORTING_WITH_CODE_OF_CONDUCT.VERSIONS[0],
+        MODERN_SLAVERY: {
+          WILL_ADHERE_TO_ALL_REQUIREMENTS: getDeclarationContentStringsByVersionId(
+            MODERN_SLAVERY[WILL_ADHERE_TO_ALL_REQUIREMENTS],
+            MODERN_SLAVERY_VERSIONS[0].WILL_ADHERE_TO_ALL_REQUIREMENTS,
```

Comment:
should this be 0 or be length - 1?

Reply:
We could use `.length - 1` but then we'd be essentially copying the implementation which I don't think is wise when testing "get the latest declaration version".